controller/server/main.go

```diff
@@ -44,6 +47,7 @@ var (
 func init() {
 	if home := homedir.HomeDir(); home != "" {
 		defaultKubeCfg = filepath.Join(home, ".kube", "config")
+		defaultTopoBasePath = filepath.Join(home, "kne-internal", "examples")
```

Comment:
Should use "kne" instead of "kne-internal" here as the default

We should not make the assumption that the user has kne-internal cloned and instead rely on "kne".

For our use case I'll specify the full path of the config

Reply:
Done. Also revised the code so it won't add this base path when checking a config file.